### ISSUE TITLE
"%matplotlib notebook" Download is Noise After Interacting with Plot

### ISSUE THREAD

**Issue by TylerHorvath** [matplotlib/matplotlib#17807]
### Bug report

**Bug summary**

After using the "%matplotlib notebook" magic command and interacting with the plot using "Pan" or "Zoom to Rectangle" the "Download Plot" button produces a image comprising of static. Before interaction it downloads the image as expected. I will note that in my browser the plot is actually subtle white and grey noise in place of the black as seen below

**Code for reproduction**

```python
%matplotlib notebook
import numpy as np
import matplotlib.pyplot as plt
x = np.linspace(0,2*np.pi,50)
plt.plot(x,np.sin(x))
plt.show()
```

**Actual outcome**

Download Plot without interaction:
![image](https://user-images.githubusercontent.com/37078950/86166529-c97fa500-bac9-11ea-9ee5-c4e9c5d14b79.png)

Download Plot after using "Pan" only
![image](https://user-images.githubusercontent.com/37078950/86169868-25006180-bacf-11ea-8554-005943ad8681.png)

Download plot after using "Zoom to Rectangle" to zoom in on x from 0 to pi
![image](https://user-images.githubusercontent.com/37078950/86166792-398e2b00-baca-11ea-919d-a362a11d748b.png)

Download plot after using "Pan" THEN "Zoom to Rectangle"
![image](https://user-images.githubusercontent.com/37078950/86169606-c0450700-bace-11ea-9640-f73799de7ab6.png)

Download plot after using "Zoom to Rectangle" THEN"Pan"
![image](https://user-images.githubusercontent.com/37078950/86169535-a4416580-bace-11ea-9a84-c1786a92e7a7.png)

**Expected outcome**
Not static, for example using "Zoom to Rectangle" to zoom in on x from 0 to pi
![image](https://user-images.githubusercontent.com/37078950/86168050-6f341380-bacc-11ea-9a11-752f89d50ec3.png)

**Matplotlib version**

First System:
  * Operating system: Linux Kernel 3.10.0-862.el7.x86_64
  * Matplotlib version: 3.1.3
  * Matplotlib backend (nbAgg):
  * Python version: 3.7.6
  * Jupyter version (if applicable): 6.0.3
  * Browser : Firefox
  * Other libraries: 

Second System:
  * Operating system:Windows 10 18363.900 machine
  * Matplotlib version: 3.2.2
  * Matplotlib backend (nbAgg):
  * Python version: 3.7.4
  * Jupyter version (if applicable): 6.0.3
  * Browser: Firefox
  * Other libraries: 

To be honest I'm not incredibly sure how the Linux system was set up since I am mostly just using it remotely (both with X-forwarding and using remote kernels), but for the Windows system I install and update everything using Anaconda or conda commands in cmd.

I haven't been able to find anything to fix this issue, and I've tried changing the backend but I'm not familiar enough with those to mess around with them, hopefully theres some kind of solution directly within the code or through upgrading/downgrading some packages. Thanks!

**Comment by tacaswell** [comment]
Interesting.  To try in improve performance what we send between the font end and the kernel is diffs between subsequent views (long runs of 0 compress really well!) so what you are seeing is the diff not static (which is still not right, but at least explainable).

Can you try with the 3.3.0rc1 release (`pip install --upgrade --pre matplotlib`)?  I can not reproduce this with firefox with 3.3.0rc1  but can reproduce it with 3.2.2.

**Comment by tacaswell** [comment]
I am inclined to close this as fixed (likely as part of @QuLogic 's overhaul to remove jquery), but would like someone else to confirm.

**Comment by tacaswell** [comment]
On second thought, to make the logistics easier I am going to close this issue.

@TylerHorvath Please ping to have it re-opened if the RC does not fix it for you!

**Comment by QuLogic** [comment]
This would have been fixed by #17027.
